File: 1.FDP/2° Competencia/C++/Repeticion/Ejercicio1.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(){
	cout << "Calcular el cambio de una compra.\n";
	double Total, Pago, matriz[i][1], Totalcambio;
	int respuesta;
	cout << "Cuantas veces deceas que se repita: ";
	cin >> respuesta;
	for ( int i=0; i<respuesta; i++) {
		cout << "Ingresar el total:$";
		cin >> Total;
		if (Total > 0){
			cout << "Ingresar el pago:$";
			cin >> Pago;
			if (Pago >= Total ) {
				matriz[i][1]= Pago-Total;
				cout << "Cambio = $" << matriz[i][1] << "\n";	
				cout << "//////////////////////////////////.\n";
			} else{ 
			cout << "Error en el pago.\n";
			cout << "///////////////////////.\n";
			}
		} else {
			cout << "Error en el total.\n";
			cout << "///////////////////////.\n";
		}
		Totalcambio= matriz[i][1]++;

		cout << Totalcambio;
	}
	
	
	return 0;
}

File: 1.FDP/2° Competencia/Java/Repeticion/Ejercicio1.java
import java.util.Scanner;	

public class Ejercicio1 {
	public static void main(String[] args){
		Scanner scan= new Scanner(System.in);
		for (int i=0; i<3; i++) {
			System.out.println("Calcular el cambio de una compra.\n");
			double Total, Pago, Cambio;
			System.out.println("Ingresar el total de compra:");
			Total= scan.nextDouble();
			if (Total > 0) {
				System.out.println("Ingresar la cantidad de pago:");
			Pago= scan.nextDouble();
				if (Pago >= Total) {
					Cambio= Pago-Total;
					System.out.println("Cambio= " + Cambio);
					System.out.println("//////////////////////////");
				}else{
					System.out.println("Error en el pago");
					System.out.println("//////////////////////////");
				}
			}else{
				System.out.println("Error en el total.\n");
				System.out.println("//////////////////////////");
			}
		}
		
	}
}